### PR TITLE
World Tomato Forum All Content

### PR DESCRIPTION
**Pull request by huseyinbiyik**
This pull request includes:

- Mobile responsive website
- Mobile responsive navigation menu
- Desktop responsive website (min-width is 768px)
- Desktop responsive menu
- Desktop responsive toolbar with social media and language links
- Speakers section which dynamically loaded with javaScript objects
- Images, logos and visual assets found on free license websites
- Created text content in every section
- A presentation video of the project
- Readme file of the project
- Scss file for the who needs

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+  .welcome {
+    margin-top: 106px;
+    height: calc(100vh - 106px);
+  }
+
+  .welcome .welcome-container {
+    text-align: left;
+    width: 100%;
+  }
+
```
Comment (Wes-Isaac) [inline style.css]: - [x] Great job so far. However, in the design, this section has spacing before and after the section, please fix that.


### Your Implementation

image

### Behance Design

image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  <footer>
+    <div class="container footer-container">
+      <img id="footer-logo" src="assets/world_tomato_forum_logo.svg" alt="">
+      <p class="small-footer-text">2023 World Tomato Forum. All Rights Reserved. <br><span class="footer-text-desktop">
+        세계 토마토 포럼은 세계 포럼의 기초입니다. 저작권 문제는 이메일 주소로 문의하시기 바랍니다.
+        </span></p>
+
+    </div>
+  </footer>
```
Comment (Wes-Isaac) [inline about.html]: - [x] Great job so far, In the design the about page, desktop version, has different styling and layout than the home page, please fix that and also the about page doesn't contain the partner section.
- [x] the background color should be **#272a31**, kindly fix that.

### Behance Design
image

### Your Implementation
image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @huseyinbiyik, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

almost-there

## Suggested changes


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 2 above)
Comment (huseyinbiyik) [reply about.html:161]: Great job so far, In the design the about page, desktop version, has different styling and layout than the home page, please fix that and also the about page doesn't contain the partner section.
-Kindly check the design. It has a partners section on the mobile version on the about page.

**Review by Lordkaito — APPROVED** [review]
Hi @huseyinbiyik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**